public/js/app/getClans.js

```diff
@@ -19,8 +19,8 @@ async function getClan() {
 
   // We compare the url TAG with the TAGS available in getAllClans and find the clan leader this way
   
-  
-  const response = await fetch(`https://api.test.faforever.com/data/clan?include=memberships.player&filter=tag==${clanTag}`);
+  // TODO: Change this hardcoded url into something with env
```

Comment:
we already have process.env.API_URL so should use that here now

Reply:
I don't think we can access env variables from the front end. This needs to be re-done with the backend rather than the frontend (which is now possible with my local fullstack enviroment thank god)

Reply:
ah got it didn't realize this was frontend my bad